common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -10,6 +10,9 @@ import play.api.libs.json.{JsArray, JsString, JsValue}
 
 object ImageMetadataConverter extends GridLogging {
 
+  // elasticsearch does not like dates with years other than 4 digits long
+  val earliestSensibleDate: DateTime = DateTime.parse("1000-01-01T00:00:00.000Z")
```

Comment:
ISO 8601 dates should always have 4 digits, the year before 1000 should be 0999

Reply:
But if you really want to minimise the chance of ES ever seeing a date with a year of 999 or lower, you might want to go with 2nd January 1000 to avoid being bitten by timezones in future.

Reply:
thanks! that makes a lot more sense than what I had been thinking. With that, I think it's probably sensible to restrict to years 1AD and later. does that also make sens to you?

Reply:
Sounds reasonable to me